### ISSUE TITLE
Suggestion:folder scan

### ISSUE THREAD

**Issue by huzpsb** [Rikonardo/PluginScan#2]
This project is great!Thank you.But I'm wondering if you can add a "folder-scan" feature to the web interface.It would be useful to detect a mass of plugins from i.e server setups.Also it would be even better if it can sort the processed files by threat level.

**Comment by Rikonardo** [comment]
Yes, I plan to add folder scanning to the web version, but since scanning in the browser is relatively slow, this is not a priority right now. Folder scanning functionality is available in the CLI version.
By the way, the ability to sort by threat level sounds interesting, I'll think about it

**Comment by huzpsb** [comment]
> Yes, I plan to add folder scanning to the web version, but since scanning in the browser is relatively slow, this is not a priority right now. Folder scanning functionality is available in the CLI version. By the way, the ability to sort by threat level sounds interesting, I'll think about it

Well,I find that to enter the command is in fact slower than scanning.In this case,can you add a gui to the stand-alone version?Thanks!
Threat level can be alike vls in anticheats.Also you can divide the level by numbers of classes or size of the file.So it would suit all sizes of plugins.
